src/viam/ur/module/ur_arm.cpp

```diff
@@ -409,23 +408,18 @@ void URArm::configure_(const std::unique_lock<std::shared_mutex>& lock, const De
     VIAM_SDK_LOG(info) << "appdir" << current_state_->appdir;
 
     // If the config contains `csv_output_path`, use that, otherwise,
-    // fall back to `VIAM_MOUDLE_DATA` as the output path, which must
+    // fall back to `VIAM_MODULE_DATA` as the output path, which must
     // be set.
     try {
-        current_state_->output_csv_dir_path = find_config_attribute<std::string>(cfg, "csv_output_path");
+        current_state_->output_csv_dir_path = find_config_attribute<std::string>(cfg, "csv_output_path").value();
+    } catch (std::invalid_argument& e) {
```

Comment:
would rather use value_or but it's argument is eagerly evaluated

Reply:
I don't love deref'ing the empty optional with `value` and then catching the resulting exception. Maybe instead this:
```
current_state_->output_csv_dir_path = find_config_attribute<std::string>(cfg, "csv_output_path");
if (!current_state_->output_csv_dir_path) {
  // deal with VIAM_MODULE_DATA
}
```
That'll still throw if the argument doesn't convert.